### ISSUE TITLE
Theme is not dark anymore after last update NC18

### ISSUE THREAD

**Issue by elpraga** [mwalbeck/nextcloud-breeze-dark#129]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Setup (please complete the following information):**
 - Nextcloud version: [e.g. 17.0.3]
 - Theme version [e.g. 17.0.6]
 - Browser and browser version [e.g. Firefox 70, Chrome 72]

**Additional context**

**Comment by mwalbeck** [comment]
Hi,

I've made a new change to allow for individual user to enable or disable the theme independently of each other. I've also decided to have the theme disabled by default going forward. I tried to have it enabled by default for old users but it sadly didn't work out. I have added a usage section to the readme with the new options for enabling the theme, https://github.com/mwalbeck/nextcloud-breeze-dark#usage

Let me know if you have any further questions
